docs/using/running.md

```diff
@@ -107,6 +107,50 @@ docker rm notebook
 # notebook
 ```
 
+## Using the Podman CLI
+
+An alternative to using the Docker CLI is to use the Podman CLI. Podman is mostly compatible with Docker.
```

Comment:
Let's add a note section with a brief description of current issues and then I think we are good to go.

Reply:
The example can be seen here: https://raw.githubusercontent.com/jupyter/docker-stacks/master/docs/maintaining/tasks.md